ui/features/plugins/core/PluginService.test.ts

```
@@ -0,0 +1,1142 @@
+    expect(s.service.getPluginState('A')?.phase).toBe('error');
+    await expect(s.service.reload('A')).rejects.toThrow(/not in ready state/);
+  });
+
+  it('#10 reload on non-existent throws', async () => {
+    await expect(s.service.reload('unknown')).rejects.toThrow(/not found/);
+  });
+
+  it('#11 loadedAt updated on successful reload', async () => {
+    s.importer.register('A', validModule());
+    await s.service.load('A');
+    const loadedAt1 = s.service.getPluginState('A')?.loadedAt!;
```

Biome (2.4.6)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)